src/ontology/cl-edit.owl

```diff
@@ -27678,6 +27674,15 @@ SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P32848))
 
+# Class: obo:CL_4023040 (L2/3-6 intratelencephalic projecting glutamatergic cortical neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2020.10.19.343129") obo:IAO_0000115 obo:CL_4023040 "A intratelencephalic-projecting glutamatergic neuron in which its soma is found in cortical layers L2/3-6")
```

Comment:
"An intratelencephalic-projecting glutamatergic neuron ~~in which its~~ with a soma ~~found~~ in cortical layers L2/3-6"